### PR TITLE
Api provides users to see upcoming trips

### PR DESCRIPTION
**Pull request by emtalen**
### User Story
```
As a resident 
In order view upcoming shopping trips and times
I would like to be able to see who and when will be shopping
```
## Changes proposed in this pull request:
* Adding index method to Pings controller only responding with active pings
* Adding active model serializer to sanitize the reponse
* Modifying the format of how the time is formatted in the response

## What I have learned working on this feature:
* How to modify the format of a datetime datatype 

## Packages/Gems added
- active model serializer

### CODE REVIEW

**Review by kfrostare — APPROVED** [review]
Very clean PR, no comments from me /Kf

File: Gemfile.lock [hunk 1]
```diff
@@ -61,6 +66,8 @@ GEM
       msgpack (~> 1.0)
     builder (3.2.4)
     byebug (11.1.1)
+    case_transform (0.2)
+      activesupport
```
Comment (johanbounce) [inline Gemfile.lock:70]: why?

**Comment by johanbounce** [comment]
Looks good!

**Comment by johanbounce** [comment]
I have a comme... oh nevermind

**Pull request merged**